docs/src/main/asciidoc/security-oidc-bearer-token-authentication.adoc

```diff
@@ -782,7 +818,7 @@ quarkus.oidc.public-key=MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAlivFI8qB4D0y
 smallrye.jwt.sign.key.location=/privateKey.pem
 ----
 
-copy `privateKey.pem` from the `integration-tests/oidc-tenancy` in the `main` Quarkus repository and use a test code similar to the one in the `Wiremock` section above to generate JWT tokens. You can use your own test keys if preferred.
+copy link:https://github.com/quarkusio/quarkus/tree/main/integration-tests/oidc-tenancy/src/main/resources/privateKey.pem[privateKey.pem] from the `integration-tests/oidc-tenancy` in the `main` Quarkus repository and use a test code similar to the one in the `Wiremock` section above to generate JWT tokens. You can use your own test keys if preferred.
```

Comment:
Thanks for adding the link. Just small note it should look something like `link:{quickstarts-tree-url}/integration-tests/oidc-tenancy/src/main/resources/privateKey.pem[privateKey.pem]`

Reply:
@jedla97 Unfortunately this resource is only available in the main repository, not in quickstarts. This doc is a concept/reference doc, so it is not strictly associated with the quickstart

Reply:
Oh sorry how i was jumping between guides I miss that this link to Quarkus repo. What I had tried now is this `link:{quarkus-blob-url}/integration-tests/oidc-tenancy/src/main/resources/privateKey.pem[privateKey.pem]` and it should work.

Reply:
Thanks @jedla97 Sure, let me do it